oracle/price-feeder/config/config.go

```diff
@@ -38,7 +38,7 @@ var (
 	ErrEmptyConfigPath = errors.New("empty configuration file path")
 
 	// SupportedProviders defines a lookup table of all the supported currency API
```

Comment:
Validators that provide data for Sei's native oracle pull price data from multiple providers. This lookup table is used to track which price feeds each provider tracks